images/base/Dockerfile

```diff
@@ -95,14 +95,16 @@ RUN echo "Ensuring scripts are executable ..." \
     && runc --version \
     && systemctl enable containerd \
  && echo "Installing crictl ..." \
-    && curl -fSL "https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz" | tar xzC /usr/local/bin \
+    && export CRICTL_URL="https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz" \
+    && curl -sSL --retry 5 --output /tmp/crictl.tgz "${CRICTL_URL}" \
+    && tar -C /usr/local/bin -xzvf /tmp/crictl.tgz \
+    && rm -rf /tmp/crictl.tgz \
  && echo "Installing CNI binaries ..." \
-    && export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/' | sed 's/armhf/arm/') \
```

Comment:
we shouldn't have a "step" depend on the contents of another step.
if we're going to dedupe this, we should move it much earlier.

Reply:
Sounds good, I can move this statement up.